app/assets/javascripts/oxalis/view/right-menu/comment_tab/comment_tab_view.js

```diff
@@ -85,6 +87,7 @@ type CommentTabStateType = {
 
 class CommentTabView extends React.PureComponent<Props, CommentTabStateType> {
   listRef: ?List;
+  storePropertyUnsubscribers: Array<Function>;
```

Comment:
`Function` is a very generic flow type and should be avoided when possible, since it lacks parameter and return types. Just write `() => void` instead.